policyengine_us/parameters/gov/states/sc/tax/income/subtractions/retirement/cap.yaml

```diff
@@ -0,0 +1,31 @@
+description: South Carolina caps the retirement deduction at this amount, based on age. 
+
+meradata:
+  threshold_unit: year
+  amount_unit: currency_USD
+  type: single_amount
+  reference:
+    - title: South Carolina State SECTION 12-6-1170(B)
+      href: https://www.scstatehouse.gov/code/t12c006.php
+    - title: South Carolina State SECTION 12-6-1170(A)(1)
+      href: https://www.scstatehouse.gov/code/t12c006.php
+    - title: South Carolina State SECTION 12-6-1170(A)(1) 
+      href: https://www.scstatehouse.gov/code/t12c006.php
+    - title: Instructions for Form SC-1040 (line 3p)
+      href: https://dor.sc.gov/forms-site/Forms/IITPacket_2021.pdf#page=16
+    - title: SC-1040 2021
+      href: https://dor.sc.gov/forms-site/Forms/SC1040_2021.pdf#page=2
+    - title: SC-1040 2022
+      href: https://dor.sc.gov/forms-site/Forms/SC1040_2022.pdf#page=2
+  period: year
+  label: South Carolina retirement deduction age threshold
+
+brackets:
+  - threshold:
+      2021-01-01: 0
+    amount:
+      2021-01-01: 3_000
+  - threshold:
+      2021-01-01: 65
+    amount: 
+      2021-01-01: 7_000
```

Comment:
and add a test
```suggestion
      2021-01-01: 10_000
```

Reply:
If I set it to 10,000, the cap amount became 13,000 instead of 10,000. Also, I need to set the age threshold in the cap.yaml 64, because the law indicates that age 65 or above can have cap amount of 10,000.

Reply:
Never mind, problem solved!